setup.py

```diff
@@ -57,6 +57,8 @@
     "yacs>=0.1.7",
     "pwlf",
     "xlsxwriter",
+    "prettytable",
```

Comment:
Not only examples need them. The core functions need these libraries, should be listed in core dependencies